spec/models/miq_region_remote_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "spec_helper"
+
+RSpec.describe MiqRegionRemote do
+  describe ".validate_connection_settings" do
+    it "returns a message about indicating the port is missing when blank" do
```

Comment:
didn't add a complimentary test for the happy path here because it required either stubbing the SUT, or nested stubs for Rails `establish_connection` stuff